data/brands/shop/hairdresser.json

```diff
@@ -66,6 +66,20 @@
         "shop": "hairdresser"
       }
     },
+    {
+      "displayName": "BrainWash",
+      "id": "brainwash-7f2667",
+      "locationSet": {"include": ["nl"]},
+      "preserveTags": ["^name"],
+      "tags": {
+        "brand": "BrainWash",
+        "brand:wikidata": "Q114905133",
+        "female": "yes",
```

Comment:
Maybe those 2 gender related tag don't need to add, as many hairdresser don't restrict customer's gender.

For example, if this is a male only barbershop, we can add a `male=only` because this tag is difinitely needed, like we treat `vegan=only` and `vegan=yes`.